pkg/apis/operator/v1/types.go

```diff
@@ -66,6 +66,12 @@ type InstallationSpec struct {
 	// NodeMetricsPort specifies which port calico/node serves metrics on. If omitted, then metrics are disabled.
 	// +optional
 	NodeMetricsPort *int32 `json:"nodeMetricsPort,omitempty"`
+
+	// FlexVolumePath optionally specifies a custom path for FlexVolume. If not specified, FlexVolume will be
+	// enabled by default. If set to 'None', FlexVolume will be disabled. The default is based on the k8s
+	// provider.
+	// +optional
```

Comment:
I think we can add a kubebuilder tag here that will further validate, so users get errors from kubectl.

Something like this probably (you'll need to figure out the right regex to represent the valid options for this).

```
// +kubebuilder:validation:Pattern="None|^\/"
```

More info here: https://book.kubebuilder.io/reference/generating-crd.html